### ISSUE TITLE
Takes forever to load if at all

### ISSUE THREAD

**Issue by ghost** [yuliskov/SmartTube#495]
I installed an update earlier yesterday and since then SmartTube has taken upwards of 5 minutes to load anything. Then if I try playing a playlist it won't let me skip to other episodes until it "loads data" which also takes about 5 minutes. Anyone have any suggestions?

**Comment by f1vefour** [comment]
I've been having this issue as well, I thought it may be sponsorblock but after disabling the issue remains.

**Comment by dlorde** [comment]
Yes, I get this too - on NVIDIA Shield TV - I just replaced Smart YouTube TV with Smart Tube Next because it took minutes to load - showing the 'buffering' symbol. Now Smart Tube Next does the same. Eventually, I get to the video grid, but if I select a video, it shows a blank screen. 

The standard YouTube app loads and plays videos in seconds, apps and all. 

Network speed isn't a problem - Ethernet at 60Mbps.

**Comment by f1vefour** [comment]
It's not that bad for me, sometimes videos just take 5 to 10 seconds to start.

If you were getting it on Smart YouTube TV also I have a feeling it's on the Google side of things.

**Comment by dlorde** [comment]
Something weird is happening; I found that if I switched to wifi on Shield TV (from the same mesh wifi access point I get the Ethernet connection from), Smart Tube Next worked perfectly the first time I played a video. But then it went back to not working... It's some kind of one-time issue because, after that, it didn't work on either Ethernet or wireless ¯\_(ツ)_/¯

**Comment by f1vefour** [comment]
I'm not using mesh, just the AC wireless on my TV and my internet is
gigabit so it's not speed related.

Without logging there is no way to know what is going on.

**Comment by dlorde** [comment]
OK, well both Smart YouTube TV and Smart Tube Next started working normally yesterday, for no apparent reason...  ¯\_(ツ)_/¯

**Comment by ACR-Jeff** [comment]
Marking as Stale
